config/pmd.xml

```diff
@@ -162,12 +162,7 @@
       <property name="problemDepth" value="4"/>
     </properties>
   </rule>
-  <rule ref="rulesets/java/design.xml/SwitchDensity">
-    <properties>
-      <!-- till #973 -->
-      <property name="violationSuppressXPath" value="//ClassOrInterfaceDeclaration[@Image='DeclarationOrderCheck']"/>
-    </properties>
-  </rule>
+  <rule ref="rulesets/java/design.xml/SwitchDensity"/>
```

Comment:
I think this row can be entirely removed - rule will be enabled with default config anyway.